server/bin/pbench-audit-server.sh

```diff
@@ -53,10 +53,10 @@
 # load common things
 . $dir/pbench-base.sh
 
-test -d $ARCHIVE || doexit "Bad ARCHIVE=$ARCHIVE"
```

Comment:
Hmm. `test -d` succeeds, which I presume is what motivated the change. However, `test -d ""` fails, so maybe just adding the quotes without the separate `-n` term would have been enough?